NotesBackEnd/tests/Notes.Api.Tests/Utils/NoteEqualityComparer.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using Notes.Api.Model;
+
+namespace Notes.Api.Tests.Utils
+{
+    class NoteEqualityComparer : IEqualityComparer<Note>

```

Comment:
State the modifier explicitly here as well.

Reply:
Also, consider sealing the class when you expect nobody to extend it.

Note: It is a must with public API to decide whether or not a class can be derived as it adds to maintenance cost. In other code, sealing is more about expressing the purpose to other programmers - a little warning that given class is not meant for an extension (in the inheritance way of the meaning).